lib/multiSemanticRelease.js

```diff
@@ -180,10 +239,29 @@ async function releasePackage(pkg, createInlinePlugin, multiContext) {
 
 	// Call semanticRelease() on the directory and save result to pkg.
 	// Don't need to log out errors as semantic-release already does that.
-	pkg.result = await semanticRelease(options, {
+	pkg.result = semanticRelease(options, {
```

Comment:
by not awaiting here, we allow parallel releases of packages within a given batch; this does mean an interleaved log at the moment, but does mean that you should have faster releases.